pkg/clean/clean_lb_infra.go

```diff
@@ -317,25 +317,38 @@ func (s *LBInfraCleaner) cleanupLBAppProfiles(ctx context.Context) error {
 	}
 	s.log.Info("Cleaning up lbAppProfiles", "Count", len(lbAppProfiles))
 	var delErr error
+	successCount := 0
+	failedCount := 0
 	for _, lbAppProfile := range lbAppProfiles {
 		select {
 		case <-ctx.Done():
+			s.log.Info("LbAppProfile cleanup interrupted by context", "successCount", successCount, "failedCount", failedCount)
 			return errors.Join(nsxutil.TimeoutFailed, ctx.Err())
 		default:
 			id := *lbAppProfile.Id
+			name := ""
+			if lbAppProfile.DisplayName != nil {
+				name = *lbAppProfile.DisplayName
+			}
+			profileType := lbAppProfile.ResourceType
+			s.log.Info("Attempting to delete LB app profile", "profileID", id, "profileName", name, "profileType", profileType)
 			if err := s.NSXClient.LbAppProfileClient.Delete(id, &forceDelete); err != nil {
-				s.log.Error(err, "Failed to deleted NCP created lbAppProfile", "lbAppProfile", id)
+				s.log.Error(err, "Failed to delete LB app profile", "profileID", id, "profileName", name, "profileType", profileType, "status", "failed")
+				failedCount++
```

Comment:
How about unify the word in the log?
some of log using 'delete', some of log using 'cleanup'
some of log using 'Attempting', the other do not use it

s.log.Error(err, "Failed to delete LB app profile", "profileID", id, "profileName", name, "profileType", profileType, "status", "failed")
it's a err log, and log 'Failed to delete LB app profile',  "status", "failed" are redundant.